src/core_init_atmosphere/mpas_init_atm_static.F

```diff
@@ -213,11 +215,13 @@ subroutine init_atm_static(mesh, dims, configs)
 
  call mpas_pool_get_config(configs, 'config_geog_data_path', config_geog_data_path)
  call mpas_pool_get_config(configs, 'config_landuse_data', config_landuse_data)
+ call mpas_pool_get_config(configs, 'config_soilcat_data', config_soilcat_data)
```

Comment:
It looks like the definition of `config_soilcat_data` is missing from the `Registry.xml` file.

Reply:
Mike, sorry about that. I added config_soilcat_data to the Registry.xml

Reply:
Thanks!